lepton/gcp.go

```diff
@@ -450,6 +450,40 @@ func (p *GCloud) ListInstances(ctx *Context) error {
 	return nil
 }
 
+// GetInstances return all instances on GCloud
+func (p *GCloud) GetInstances(ctx *Context) ([]compute.Instance, error) {
```

Comment:
this needs to be called from ListInstances and is missing quite a few things like pub/priv ip, created_at date, status, etc..

that could/should be a common struct that is re-used amongst all the various providers